roofit/xroofit/src/xRooNode_interactive.cxx

```diff
@@ -85,6 +89,8 @@ void xRooNode::Interactive_Pull()
             } else {
                select->SetPointX(0, _gr->GetPointX(i - 1));
             }
+            dynamic_cast<TGraph *>(dynamic_cast<TMultiGraph *>(gPad->GetPrimitive("editables"))
```

Comment:
```suggestion
            static_cast<TGraph *>(static_cast<TMultiGraph *>(gPad->GetPrimitive("editables"))
```
The overhead of `dynamic_cast` is only worth it if you check if the cast succeeded by checking if the result was `nullptr`. Here you are using the return value directly without validating it, so you can just do `static_cast`.